cryptoConsole/cryptowatch.py

```diff
@@ -21,84 +21,105 @@
+         Created by: Alex Epstein https://github.com/alexanderepstein'''
 
 
-config = cfg.config()
+def print_header():
+    print(HEADER)
 
-def printHeader():
-    print('_________                        __                         __         .__')
-    print('\_   ___ \_______ ___.__._______/  |_  ______  _  _______ _/  |_  ____ |  |__')
-    print('/    \  \/\_  __ <   |  |\____ \   __\/  _ \ \/ \/ /\__  \\\\   __\/ ___\|  |  \\')
-    print('\     \____|  | \/\___  ||  |_> >  | (  <_> )     /  / __ \|  | \  \___|   Y  \\')
-    print('\______  /|__|   / ____||   __/|__|   \____/ \/\_/  (____  /__|  \___  >___|  /')
-    print('       \/        \/     |__|                             \/          \/     \/')
-    print("         Created by: Alex Epstein https://github.com/alexanderepstein")
 
-def cryptoFile(filePath):
-    if platform in ["linux", "linux2", "darwin"]:
-        if "~" in filePath:
-            filePath = sub("~", expanduser("~"), filePath)
-    if exists(filePath):
-        answer = input("File already exists at %s, overwrite it? [Y/n] " % filePath)
-        answer = answer.lower()
-        if answer != 'y' and answer != "yes":
+def crypto_file(file_path):
+    if platform in ("linux", "linux2", "darwin") and "~" in file_path:
+        file_path = sub("~", expanduser("~"), file_path)
+    if exists(file_path):
+        answer = input("File already exists at %s, overwrite it? [Y/n] "
+                       % file_path)
+        if answer.lower().strip() not in ('y', 'yes'):
             exit()
     try:
-        with open(filePath, 'w+') as file:
+        with open(file_path, 'w+') as file:
             data = getCryptoTable(False, False)
             file.write(data)
             file.write("\n")
     except IsADirectoryError:
         print("Error: the path provided is a directory")
         exit()
-    printHeader()
+    print_header()
 
 
-def consoleLoop():
+def console_loop():
     try:
         while True:
             print(getCryptoTable(True))
             sleep(30)
     except KeyboardInterrupt:
         clear()
-        printHeader()
+        print_header()
+
 
 def main():
-    parser = argparse.ArgumentParser(prog="Cryptowatch",description='Track prices and account balances for bitcoin, ethereum, and litecoin', epilog="By: Alex Epstein https://github.com/alexanderepstein")
-    parser.add_argument("-m", "--monitor",help="Choose which cryptowatch monitor to use")
-    parser.add_argument("-f", "--file", default="", help="Output the current cryptowatch data to the specified file path")
-    parser.add_argument("-c", "--config", action = "store_true", help="Edit the config file for cryptowatch")
-    parser.add_argument("-v", "--version", action="store_true", help="Display the current version of cryptowatch")
+    parser = argparse.ArgumentParser(
+        prog="Cryptowatch",
+        description='Track prices and account balances for bitcoin, '
```

Comment:
Can you also add that it tracks bitcoin cash and dash here as well (this was incorrect even before you made your PR)

Reply:
@alexanderepstein ,

Ah ok, I'm on it.